website/api_versioned_docs/version-1.0/blockchain-integration/contracts/verifier.md

```diff
@@ -47,16 +47,33 @@ You can find detailed information in the [version management design][VersionMana
+|                    | Sepolia | [pubkey](https://sepolia.arbiscan.io/address/0x3a54A45E44a71020Bd0Af42063B9f23e8b9E387D)                 | [contract](https://sepolia.arbiscan.io/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                   | [contract](https://sepolia.arbiscan.io/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                   |
+| Avalance (C-Chain) | Mainnet | [pubkey](https://avascan.info/blockchain/all/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)         | [contract](https://avascan.info/blockchain/c/address/0xDC986a09728F76110FF666eE7b20d99086501d15/contract)         | [contract](https://avascan.info/blockchain/c/address/0x0b144E07A0826182B6b59788c34b32Bfa86Fb711/contract)         |
+|                    | Fuji    | [pubkey](https://testnet.avascan.info/blockchain/all/address/0x3a54A45E44a71020Bd0Af42063B9f23e8b9E387D) | [contract](https://testnet.avascan.info/blockchain/c/address/0xDC986a09728F76110FF666eE7b20d99086501d15/contract) | [contract](https://testnet.avascan.info/blockchain/c/address/0x0b144E07A0826182B6b59788c34b32Bfa86Fb711/contract) |
+| Base               | Mainnet | [pubkey](https://basescan.org/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)                        | [contract](https://basescan.org/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                          | [contract](https://basescan.org/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                          |
+|                    | Sepolia | [pubkey](https://sepolia.basescan.org/address/0x3a54A45E44a71020Bd0Af42063B9f23e8b9E387D)                | [contract](https://sepolia.basescan.org/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                  | [contract](https://sepolia.basescan.org/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                  |
+| Linea              | Mainnet | [pubkey](https://lineascan.build/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)                     | [contract](https://lineascan.build/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                       | [contract](https://lineascan.build/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                       |
+| Optimism           | Mainnet | [pubkey](https://optimistic.etherscan.io/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)             | [contract](https://optimistic.etherscan.io/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)               | [contract](https://optimistic.etherscan.io/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)               |
+| Polygon zkEVM      | Mainnet | [pubkey](https://zkevm.polygonscan.com/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)               | [contract](https://zkevm.polygonscan.com/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                 | [contract](https://zkevm.polygonscan.com/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                 |
```

Comment:
I don't really see folks use the term "pubkey" in the Ethereum context. It's also not quite what I think of as a public key, since it's hashed and truncated (I see it as closer to a fingerprint).
```suggestion
| Chain              | Network | RISC Zero Admin Address                                                                                   | [Timelock][TimelockController.sol]                                                                                | [Router][RiscZeroVerifierRouter.sol]                                                                              |
| ------------------ | ------- | -------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
| Ethereum           | Mainnet | [address](https://etherscan.io/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)                        | [contract](https://etherscan.io/address/0x0b144E07A0826182B6b59788c34b32Bfa86Fb711#code)                          | [contract](https://etherscan.io/address/0x8EaB2D97Dfce405A1692a21b3ff3A172d593D319#code)                          |
|                    | Sepolia | [address](https://sepolia.etherscan.io/address/0x3a54a45e44a71020bd0af42063b9f23e8b9e387d)                | [contract](https://sepolia.etherscan.io/address/0xB4E3306129208cC8e6E75157f75f62eAe0B920a0#code)                  | [contract](https://sepolia.etherscan.io/address/0x925d8331ddc0a1F0d96E68CF073DFE1d92b69187#code)                  |
| Arbitrum           | Mainnet | [address](https://arbiscan.io/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)                         | [contract](https://arbiscan.io/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                           | [contract](https://arbiscan.io/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                           |
|                    | Sepolia | [address](https://sepolia.arbiscan.io/address/0x3a54A45E44a71020Bd0Af42063B9f23e8b9E387D)                 | [contract](https://sepolia.arbiscan.io/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                   | [contract](https://sepolia.arbiscan.io/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                   |
| Avalance (C-Chain) | Mainnet | [address](https://avascan.info/blockchain/all/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)         | [contract](https://avascan.info/blockchain/c/address/0xDC986a09728F76110FF666eE7b20d99086501d15/contract)         | [contract](https://avascan.info/blockchain/c/address/0x0b144E07A0826182B6b59788c34b32Bfa86Fb711/contract)         |
|                    | Fuji    | [address](https://testnet.avascan.info/blockchain/all/address/0x3a54A45E44a71020Bd0Af42063B9f23e8b9E387D) | [contract](https://testnet.avascan.info/blockchain/c/address/0xDC986a09728F76110FF666eE7b20d99086501d15/contract) | [contract](https://testnet.avascan.info/blockchain/c/address/0x0b144E07A0826182B6b59788c34b32Bfa86Fb711/contract) |
| Base               | Mainnet | [address](https://basescan.org/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)                        | [contract](https://basescan.org/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                          | [contract](https://basescan.org/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                          |
|                    | Sepolia | [address](https://sepolia.basescan.org/address/0x3a54A45E44a71020Bd0Af42063B9f23e8b9E387D)                | [contract](https://sepolia.basescan.org/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                  | [contract](https://sepolia.basescan.org/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                  |
| Linea              | Mainnet | [address](https://lineascan.build/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)                     | [contract](https://lineascan.build/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                       | [contract](https://lineascan.build/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                       |
| Optimism           | Mainnet | [address](https://optimistic.etherscan.io/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)             | [contract](https://optimistic.etherscan.io/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)               | [contract](https://optimistic.etherscan.io/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)               |
| Polygon zkEVM      | Mainnet | [address](https://zkevm.polygonscan.com/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)               | [contract](https://zkevm.polygonscan.com/address/0xdc986a09728f76110ff666ee7b20d99086501d15#code)                 | [contract](https://zkevm.polygonscan.com/address/0x0b144e07a0826182b6b59788c34b32bfa86fb711#code)                 |
```